src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -80,28 +80,48 @@ You can also apply this configuration when unsetting a user at logout:
 
 <PlatformContent includePath="enriching-events/unset-user" />
 
+<PlatformSection supported={["android"]}>
+
+<PlatformContent includePath="enriching-events/scopes/scope-synchronization" />
+
+</PlatformSection>
+
 To learn what useful information can be associated with scopes see
 [the context documentation](../context/).
 
 ## Local Scopes
 
-We also have support for pushing and configuring a scope in one go. This is typically called `with-scope` or `push-scope` which is also very helpful if you only want to send data with one specific event. In the following example we are using that function to attach a `level` and a `tag` to only one specific error:
+<PlatformSection supported={["java"]} notSupported={["android"]}>
```

Comment:
This is wrong, why is it only supported for Java?
See https://sentry-docs-git-fork-designedbyz-update-scopes.sentry.dev/platforms/apple/guides/ios/enriching-events/scopes/#local-scopes

Reply:
Ooop, I think that's just a copy paste error on my part. 

Fixed in 1d1fbe03